base/src/main/java/org/aya/core/serde/CompiledAya.java

```diff
@@ -51,7 +51,7 @@ public record CompiledAya(
   /**
    * @param rename not empty
    */
-  record SerImport(@NotNull ImmutableSeq<String> moduleName, @NotNull ImmutableSeq<String> rename,
+  record SerImport(@NotNull ModulePath.Qualified moduleName, @NotNull ImmutableSeq<String> rename,
```

Comment:
`ModulePath.Qualified` represents a **name** in a context instead of an absolute path. In fact, I think it should be renamed `ModuleName`, and `ModulePath` for absolute paths.

Reply:
Oh okay.

Reply:
Can you do this?